java/opendcs/src/main/java/opendcs/dao/DaoBase.java

```diff
@@ -235,19 +236,18 @@ public ResultSet doQuery(String q)
                 queryStmt1 = getConnection().createStatement();
             if (fetchSize > 0)
                 queryStmt1.setFetchSize(fetchSize);
-            debug3("Query1 '" + q + "'");
-
-//if (this instanceof decodes.cwms.CwmsTimeSeriesDAO
-// || this instanceof decodes.cwms.CwmsSiteDAO)
-//debug1("Fetch size=" + queryStmt1.getFetchSize());
+            log.trace("Query1 '{}'", q);
 
             return queryResults1 = queryStmt1.executeQuery(q);
         }
         catch(SQLException ex)
         {
-            String msg = "SQL Error in query '" + q + "': " + ex;
-            warning(msg);
-            throw new DbIoException(msg);
+            String msg = "SQL Error in query '" + q + "'";
+            if (log.isTraceEnabled())
+            {
+                log.atWarn().setCause(ex).log(msg);
+            }
+            throw new DbIoException(msg, ex);
```

Comment:
```suggestion
            log.atWarn().setCause(ex).log(msg);
            
            throw new DbIoException(msg, ex);
```

When would we need  if (log.isTraceEnabled()) ?   It seems simpler to just have the log.atWarn() do the filtering?